backend/routes/api/users.js

```diff
@@ -124,20 +124,14 @@ router.post(
   })
 );
 
-// set role
-router.put(
-  "/:user_id/role",
+// set role and title
+router.patch(
```

Comment:
is there a way to generalize this to be able to set any arbitrary field? just wondering

Reply:
Yes! We actually just removed that functionality since Eugenia and I discussed how we'll probably never have to change something like first name or last name since that's coming from Google OAuth, but we could absolutely keep in that functionality if you think it's better to support that.